packages/api/tests/task.test.js

```diff
@@ -2047,11 +1996,11 @@ describe('Task tests', () => {
 
   describe('Patch task date test', () => {
```

Comment:
@MwayaB I'd suggest being specific in the descriptive text that this is about "due date". A task has other kinds of date(s) associated with it.